src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -871,9 +874,35 @@ End Property
 
                 await WaitWaiterAsync(workspace.ExportProvider);
 
-                // mutate solution
-                workspace.OnSolutionAdded(solution);
+                // block solution cralwer from processing.
```

Comment:
This should _really_ be one of those "why?" comments, not "what?" comments. :smile:

Reply:
sure. I can add why we are doing this.